### ISSUE TITLE
Accessibility when UITextField floating above ASTableView

### ISSUE THREAD

**Issue by onato** [TextureGroup/Texture#2048]
I have a floating UI element that is built with UIKit. It is in an ASDKViewController added to the view. Currently, it is not accessible because Texture seems to be managing the accessibility elements and is not expecting this use case. I'm looking for the expected and supported way to implement this.

<img width="272" alt="Screen Shot 2021-12-03 at 1 13 15 PM" src="https://user-images.githubusercontent.com/107999/144523038-8f86c7a3-c7fe-4d7c-9004-1d964b57d0e7.png">

## If the UITextView has no Texture element underneath, it is accessible.
This also works with a UITableView under the UITextView.

<img width="1076" alt="Screen Shot 2021-12-03 at 1 22 23 PM" src="https://user-images.githubusercontent.com/107999/144523913-7dd6c34a-8d39-4e3a-8590-d20f45eb2665.png">

## When I add an ASTableView, the UITextView becomes inaccessable.
<img width="1081" alt="Screen Shot 2021-12-03 at 1 23 12 PM" src="https://user-images.githubusercontent.com/107999/144523899-3d96be65-580c-4766-be26-94d17b03a9ef.png">

I have created a minimal example. I would like the UITextField in ViewController.swift to be accessible with the Accessibility Inspector.

[XibTest.zip](https://github.com/TextureGroup/Texture/files/7645929/XibTest.zip)

**Comment by onato** [comment]
I managed to get it to work with a dirty hack. I replaced the root node with this one and added my `UIView` subclass to `additionalViews`.

Is there a cleaner/supported way to do this?

```
class AccessableDisplayNode: ASDisplayNode {
    var additionalViews: [UIView] = []
    override var accessibilityElements: [Any]? {
        get {
            (super.accessibilityElements ?? []) + additionalViews
        }
        set {
            super.accessibilityElements = newValue
        }
    }
}
```
